### ISSUE TITLE
Resolving Package Graph Failed

### ISSUE THREAD

**Issue by sgraesser17** [apple/swift#65961]
**Description**
Resolving Package Graph Failed

**Steps to reproduce**

**Expected behavior**
Package Graph resolved

**Environment**
- swift-driver version: 1.75.2 Apple Swift version 5.8 (swiftlang-5.8.0.124.2 clang-1403.0.22.11.100)
- Xcode 14.3 Build version 14E222b
- Deployment target: iOS 14.0

Showing All Errors Only
InternalError(description: "Internal error. Please file a bug at https://bugs.swift.org with this info. Expected root cause {apptemplatecodecomponents.common[everything] 1.2.3, ¬apptemplatecodecomponents.sniwrapper[everything] 1.0.0..<2.0.0} to almost satisfy the current partial solution:\n * [Decision 0: <synthesized-root>[everything] 1.0.0]\n * [Derivation: youtube-ios-player-helper[everything] 1.0.0..<2.0.0 ← {<synthesized-root>[everything] 1.0.0, ¬youtube-ios-player-helper[everything] 1.0.0..<2.0.0}]\n * [Derivation: sdwebimage[everything] 5.0.0..<6.0.0 ← {<synthesized-root>[everything] 1.0.0, ¬sdwebimage[everything] 5.0.0..<6.0.0}]\n * [Derivation: phonenumberkit[everything] 3.0.0..<4.0.0 ← {<synthesized-root>[everything] 1.0.0, ¬phonenumberkit[everything] 3.0.0..<4.0.0}]\n * [Derivation: messagekit[everything] 3.0.0..<4.0.0 ← {<synthesized-root>[everything] 1.0.0, ¬messagekit[everything] 3.0.0..<4.0.0}]\n * [Derivation: mbprogresshud[everything] 1.0.0..<2.0.0 ← {<synthesized-root>[everything] 1.0.0, ¬mbprogresshud[everything] 1.0.0..<2.0.0}]\n * [Derivation: haishinkit.swift[everything] 1.0.0..<2.0.0 ← {<synthesized-root>[everything] 1.0.0, ¬haishinkit.swift[everything] 1.0.0..<2.0.0}]\n * [Derivation: fscalendar[everything] 2.0.0..<3.0.0 ← {<synthesized-root>[everything] 1.0.0, ¬fscalendar[everything] 2.0.0..<3.0.0}]\n * [Derivation: facebook-ios-sdk[everything] 12.0.0..<13.0.0 ← {<synthesized-root>[everything] 1.0.0, ¬facebook-ios-sdk[everything] 12.0.0..<13.0.0}]\n * [Derivation: charts[everything] 4.0.0..<5.0.0 ← {<synthesized-root>[everything] 1.0.0, ¬charts[everything] 4.0.0..<5.0.0}]\n * [Derivation: apptemplatecodecomponents.sniwrapper[everything] 1.13.0-alpha1..<2.0.0 ← {<synthesized-root>[everything] 1.0.0, ¬apptemplatecodecomponents.sniwrapper[everything] 1.13.0-alpha1..<2.0.0}]\n * [Derivation: apptemplatecodecomponents.sirqulsdk[everything] 1.6.0..<2.0.0 ← {<synthesized-root>[everything] 1.0.0, ¬apptemplatecodecomponents.sirqulsdk[everything] 1.6.0..<2.0.0}]\n * [Derivation: apptemplatecodecomponents.common[everything] 1.2.3..<2.0.0 ← {<synthesized-root>[everything] 1.0.0, ¬apptemplatecodecomponents.common[everything] 1.2.3..<2.0.0}]\n * [Derivation: ¬apptemplatecodecomponents.common[everything] 1.2.4..<2.0.0 ← {apptemplatecodecomponents.common[everything] 1.2.4..<2.0.0}]\n")

**Comment by AnthonyLatsis** [comment]
cc @neonichu

**Comment by neonichu** [comment]
Looks like something that should be transferred over to SwiftPM.

**Comment by AnthonyLatsis** [comment]
Yeah, I was hoping you could perform the transfer. Will you please, or tag someone from SwiftPM who can?

**Comment by neonichu** [comment]
Don't think I can, cc @shahmishal
